code-reviews/recipes/Markdown.md

```diff
@@ -0,0 +1,175 @@
+```
+
+Fixing errors automatically
+
+```bash
+markdownlint **/*.md --ignore node_modules --fix
+```
+
+### proselint
+
+[`proselint`](http://proselint.com/) is a command line utility that lints the text contents of the document.  It checks for jargon, spelling errors, redundancy, corporate speak and other language related issues.
+
+It's available both as a [python package](https://github.com/amperser/proselint/#checks) and a [node package](https://www.npmjs.com/package/proselint).
+
+```bash
+pip install proselint
+npm install -g proselint
+```
+
+Run proselint
+
+```bash
+proselint document.md
+```
+
+### write-good
+
+[`write-good`](https://github.com/btford/write-good) is a linter for English text that helps writing better documentation.
+
+```bash
+npm install -g write-good
+```
+
+Run write-good
+
+```bash
+write-good *.md
+```
+
+Run write-good without installing it
+
+```bash
+npx write-good *.md
+```
+
+Write Good is also available as an [extension for VS Code](https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter)
+
+## VS Code Extensions
+
+### Write Good Linter
+
+The [`Write Good Linter Extension`](https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter) integrates with VS Code to give grammar and language advice while editing the document.
+
+### markdownlint extension
+
+The [`markdownlint extension`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) examines the markdown documents, showing warnings for rule violations while editing.
+
+## Build validation
+
+To automate linting with `markdownlint` for PR validation in GitHub actions, use the following YAML.
+
+```yaml
+name: Markdownlint
+
+on:
+  push:
+    paths:
+      - "**/*.md"
+  pull_request:
+    paths:
+      - "**/*.md"
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Run Markdownlint
+      run: |
+        npm i -g markdownlint-cli
+        markdownlint "**/*.md"
+```
+
+## Code Review Checklist
+
+Apart from the items on the [Code Review Checklist](../readme.md) you should also look for these markdown specific code review items
+
+- [ ] Is the document easy to read and understand and does it follow [good writing guidelines](#writing-style-guidelines)?
+- [ ] Is there a single source of truth or is content repeated in more than one document?
+- [ ] Is the documentation up to date with the code?
+
+## Writing Style Guidelines
+
+The following are some examples of writing style guidelines.
+
+Agree in your team which guidelines you should apply to your project documentation.
+Save your guidelines together with your documentation so they are easy to refer back to.
+
+### Wording
+
+- Use inclusive language, and avoid jargon and uncommon words. The docs should be easy to understand.
```

Comment:
nit: some lists items are using a dot at the end (Wording) others are not (Document Organziation) or are inconsistent within the list. We could be consistent here.

Reply:
Good catch :) old habit, but great that you saw it, it's a little extra important in this doc